genx/setup.cfg

```diff
@@ -1,14 +1,51 @@
-[sdist]
-formats=gztar
+[metadata]
+name = GenX
+version = attr: genx.version.__version__
+author = Matts Bjorck, Artur Glavic
+author_email = artur.glavic@gmail.com
+description = X-ray and Neutron reflectivity fitting software.
+long_description = file: README.txt
+url = http://genx.sourceforge.net/
+license = GPL v3
+license_file = LICENSE.txt
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: OS Independent
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 6 - Mature
 
-[bdist]
-formats=rpm
 
-[bdist_rpm]
-requires=python3
-packager = Artur Glavic <artur.glavic@psi.ch>
+[options]
+python_requires = >=3.6
+packages =
+    genx
+    genx.lib
+    genx.models
+    genx.models.lib
+    genx.plugins
+    genx.plugins.add_ons
+    genx.plugins.add_ons.help_modules
+    genx.plugins.data_loaders
+    genx.plugins.data_loaders.help_modules
+scripts = scripts/genx
+install_requires =
```

Comment:
There used to be a distiction between "requires" and "install_requires". The packages you list are required to run, not install genx.

Reply:
According to https://packaging.python.org/guides/distributing-packages-using-setuptools/#install-requires, install_requires specifies "what dependencies a project minimally needs to run" (as opposed to build_requires).